src/js/zqd/zqd.js

```diff
@@ -78,33 +79,29 @@ function zqdCommand(): string {
   return zqdBin
 }
 
-function zeekRunnerCommand(): string {
+function zeekRunnerCommand(zeekRunnerPref: string): string {
   const plat = platformDefs[process.platform]
   if (!plat) {
     throw new Error("unsupported platform for zqd")
   }
 
-  let zeekRunner = process.env.BRIM_ZEEK_RUNNER
-  if (!zeekRunner) {
-    // TODO: https://github.com/brimsec/brim/issues/741
-    // If the environment varible isn't set, allow the user to set
-    // a preferred zeekrunner location via preferences.
-    zeekRunner = resolve(join(zdepsDirectory, "zeek", plat.zeekRunnerBin))
-  }
-
-  if (!pathExistsSync(zeekRunner)) {
-    throw new Error("zeek runner not present at " + zeekRunner)
-  }
+  const precedence = [
```

Comment:
`s/precedence/preference` ?

Reply:
I meant precedence here. This is the lookup order for the zeek runner path. First check the ENV, then the userPref, then the default.